src/main/java/io/supertokens/inmemorydb/queries/OAuthQueries.java

```diff
@@ -163,6 +144,43 @@ public static OAuthClient getOAuthClientById(Start start, String clientId, AppId
         });
     }
 
+    public static void createOrUpdateOAuthSession(Start start, AppIdentifier appIdentifier, @NotNull String gid, @NotNull String clientId,
+                                                  String externalRefreshToken, String internalRefreshToken, String sessionHandle,
+                                                  List<String> jtis, long exp)
+            throws SQLException, StorageQueryException {
+        String QUERY = "INSERT INTO " + Config.getConfig(start).getOAuthSessionsTable() +
+                " (gid, client_id, app_id, external_refresh_token, internal_refresh_token, session_handle, jti, exp) VALUES (?, ?, ?, ?, ?, ?, ?, ?) " +
+                "ON CONFLICT (gid) DO UPDATE SET external_refresh_token = ?, internal_refresh_token = ?, " +
+                "session_handle = ? , jti = ?, exp = ?";
+        update(start, QUERY, pst -> {
+            String jtiDbValue = jtis == null ? null : String.join(",", jtis);
+
+            pst.setString(1, gid);
+            pst.setString(2, clientId);
+            pst.setString(3, appIdentifier.getAppId());
+            pst.setString(4, externalRefreshToken);
+            pst.setString(5, internalRefreshToken);
+            pst.setString(6, sessionHandle);
+            pst.setString(7, jtiDbValue);
+            pst.setLong(8, exp);
+
+            List<String> alreadySavedJTIs = getOAuthJTIsByGID(start, appIdentifier, gid);
```

Comment:
This feels weird. Couldn't we do this by some kind of string update thing? (e.g. if we were OK with just having an extra comma at the end, it could be a simple concat operation)